package.json

```diff
@@ -87,6 +89,8 @@
 	"dependencies": {
 		"jsonc-parser": "^2.1.0",
 		"tar": ">=4.4.2",
+		"tree-sitter-agda": "^1.3.1",
+		"tree-sitter-haskell": "github:tree-sitter/tree-sitter-haskell#d72f2e42c0d5ccf8e8b1c39e3642428317e8fe02",
```

Comment:
these are supposed to go in `devDependencies` like the other language packages, because the npm package is just used to build the wasm.  However, I got some weird errors when I tried to load a haskell file after I changed that.  Did you already try `devDependencies` and run into some kind of error?

Reply:
I think haskell-tree-sitter hasn’t published new versions on npm recently.

Reply:
oh ok so maybe haskell-tree-sitter doesn't work with latest tree-sitter code?

Reply:
were you able to open a haskell file?